### ISSUE TITLE
CX executable being sent to $GOPATH/bin/

### ISSUE THREAD

**Issue by amherag** [skycoin/cx#46]
As a number of users could not have $GOPATH/bin/ in their executable path, they could end up with a successful installation, but not being able to run CX from a global scope in their terminals.

We need to send CX to /usr/bin or something like that, and also find a solution for Windows.

Or maybe we could make the installation script check if $GOPATH/bin is in the $PATH?

**Comment by gz-c** [comment]
Usually I see instructions to the user to add $GOPATH/bin to their $PATH

**Comment by amherag** [comment]
Yeah, but a lot of people forget to do this or don't care at all. Binary releases help a lot because it's now responsibility of the user to place the binary wherever they want.

We could actually close this now and just assume that if they a user is brave enough to be compiling CX from source, they should know what $GOPATH is and how golang workspaces work.

**Comment by gz-c** [comment]
Installation from source instructions could include the remark about adding to gopath, that would be enough
